particles/Tutorial/Kotlin/README.md

```diff
@@ -37,12 +37,9 @@ Alright, we’ve got our Arcs manifest file set. Now onto the Kotlin. Just like
 ```kotlin
 package arcs.tutorials
 
-// We start with the required imports.
-import arcs.sdk.Particle
-
 // We define the HelloWorld class to be a child
 // of the Particle class.
```

Comment:
Update comment to say we extend the AbstractHelloWorld class.

Reply:
Done.